dangerzone/isolation_provider/qubes.py

```diff
@@ -67,6 +67,7 @@ def start_doc_to_pixels_proc(self, document: Document) -> subprocess.Popen:
             stdin=subprocess.PIPE,
             stdout=subprocess.PIPE,
             stderr=stderr,
+            start_new_session=True,
```

Comment:
I believe we should add the commit message here as a comment for clarity.

```suggestion
            # Start the conversion process in a new session, so that we can later on
            # kill the process group, without killing the controlling script.
            start_new_session=True,
```

Reply:
Makes sense, did so in dfc3dfd.